ledgerwallet/ledgerctl.py

```diff
@@ -162,9 +171,15 @@ def list_apps(get_client, remote, url, key):
     help="Delete using application hash instead of application name",
     is_flag=True,
 )
+@click.option(
+    "-d",
+    "--dump",
+    help="Dump APDU installation file.",
+    is_flag=False,
+    flag_value="out.apdu",
```

Comment:
I'm not use to click, what's the purpose of this flag_value?

Reply:
Cf. Click documentation https://click.palletsprojects.com/en/8.1.x/options/

```
Providing the value to an option can be made optional, in which case providing only the option’s flag without a value will either show a prompt or use its flag_value.

Setting is_flag=False, flag_value=value tells Click that the option can still be passed a value, but if only the flag is given the flag_value is used.
```